ygen/protogen.go

```diff
@@ -358,6 +387,37 @@ func genProto3Msg(msg *yangDirectory, msgs map[string]*yangDirectory, state *gen
 	return append(msgDefs, msgDef), errs
 }
 
+// genProtoEnum takes an input yang.Entry that contains an enumerated type
+// and returns a protoMsgEnum that contains its definition within the proto
+// schema.
+func genProtoEnum(field *yang.Entry) (*protoMsgEnum, error) {
+	eval := map[int64]string{}
+	names := field.Type.Enum.NameMap()
+	eval[0] = "UNSET"
```

Comment:
Suggest aggregating proto string literals into a const block somewhere and using those definitions instead.

Reply:
Done.